agent/ZappiehostBuyer.py

```diff
@@ -113,7 +113,10 @@ def placeOrder(self):
 
             self.driver.find_element_by_css_selector("input[type='submit'][class='cartbutton green ui-button ui-widget ui-state-default ui-corner-all']").click() # Submit the form
 
-            self.driver.find_element_by_css_selector("input[type='submit'][value='Pay Now']").click()
+            try:
+                self.driver.find_element_by_css_selector('input[value="Pay Now"]').click()
+            except Exception as e:
+                print("Warning: Pay now button not found")

```

Comment:
I think this is the definition of a pointless variable. Furthermore I highly doubt that printing "Pay button not found" is something that should happen in the code as only thing after catching an exception.